### PR TITLE
feat/new_data_cron

### PR DESCRIPTION
**Pull request by maiali13**
# HRF Police Violence DS Team Pull Request Template

Changes function imports for cron job:
- rename file to 'utilities'
- add get_new fxn which outputs new incidents
- add state_code in clean_pb2020 fxn

Also removes 'county' from place table in db_postgres.py database seed script

https://trello.com/c/u3OnEXuW

## Types of Changes

Types of changes are to be set in the pull request name as 'LABEL: description'.

FEAT/new_data_cron

## PR Creator Checklist

- [x] I wrote a label and description of the requested changes
- [x] PR was sent to multiple reviewers, including at least one team member and the TPL
- [x] I performed a self-review of my own code (followed the code review checklist)
- [x] I linked to the user story/Trello checklist item corresponding to the PR

## Code Review Checklist

Bearing in mind the [Pull Request Rubric](https://www.notion.so/1fc04e4fedeb429ba873b7c68d281707?v=74054da7991341c0bf970f39410c43da):

- [ ] Atomic, well-named commits
- [ ] PR documents the motivation, user story and changes
- [ ] Code is consistently formatted 
- [ ] PR is appropriately scoped- small or only one issue
- [ ] Code is idiomatic and conforms to industry standards
- [ ] At least one team member or the TPL has reviewed the PR and left appropriate comments
- [ ] (When applicable) Reviewer suggestions were acted upon

### CODE REVIEW

**Review by axefx — APPROVED** [review]
This looks good, cleans up the endpoint.

File: Data/db_postgresql.py [hunk 1]
```diff
@@ -113,21 +111,20 @@
...
+        (city, state_code, state_name, latitude, longitude, counter)
         VALUES %s
         """
     psycopg2.extras.execute_values(
         pg_curs, sql, params, page_size=10000)
 
 # function that inserts data into tables from training data csv
-with open('/Users/michelle/Labs25-Human_Rights_First-TeamC-DS/Data/training_data.csv', 'r') as f:
+with open('training_data.csv', 'r') as f:
     reader = csv.reader(f)
```
Comment (michhottinger) [inline Data/db_postgresql.py:128]: did you check to see if this path works? You may need to reference it differently if we move these files around.

File: Data/db_postgresql.py [hunk 2]
```diff
@@ -145,7 +142,7 @@
 
 
 # evidence: id, link
-with open('/Users/michelle/Labs25-Human_Rights_First-TeamC-DS/Data/training_data.csv', 'r') as f:
+with open('training_data.csv', 'r') as f:
     reader = csv.reader(f)
```
Comment (michhottinger) [inline Data/db_postgresql.py:146]: be sure to check this file path! it will be off if we move any of these files

**Review by michhottinger — COMMENTED** [review]
This looks good--I see you removed "county".

**Pull request merged**